package.json

```diff
@@ -10,6 +10,9 @@
     "start": "node build/server.js",
     "dev": "nodemon src/server.ts",
     "build": "tsc -p .",
+    "db:dockerstart": "echo \"🐳✨ Starting docker compose...✨🐳\" && docker compose -f docker/docker-compose-db-dev.yml up",
+    "db:dockerstop": "echo \"🐋 Stopping docker compose... 🐋\" && docker compose -f docker/docker-compose-db-dev.yml down",
```

Comment:
I'd suggest naming these scripts using docker's up/down convention for compose, rather than start/stop which is used for containers.

Reply:
Done ✅